sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/base_node.py

```diff
@@ -66,31 +78,41 @@ class BaseNode(Job, PipelineNodeIOMixin, YamlTranslatableMixin, _AttrDict, Schem
     You should not instantiate this class directly. Instead, you should
     create from a builder function.
 
-    :param type: Type of pipeline node
+    :param type: Type of pipeline node. Defaults to JobType.COMPONENT.
     :type type: str
     :param component: Id or instance of the component version to be run for the step
-    :type component: Union[Component, str]
-    :param inputs: Inputs to the node.
-    :type inputs: Dict[str, Union[Input, SweepDistribution, str, bool, int, float, Enum, dict]]
-    :param outputs: Mapping of output data bindings used in the job.
-    :type outputs: Dict[str, Union[str, Output, dict]]
-    :param name: Name of the node.
-    :type name: str
-    :param description: Description of the node.
-    :type description: str
-    :param tags: Tag dictionary. Tags can be added, removed, and updated.
-    :type tags: dict[str, str]
-    :param properties: The job property dictionary.
-    :type properties: dict[str, str]
-    :param comment: Comment of the pipeline node, which will be shown in designer canvas.
-    :type comment: str
-    :param display_name: Display name of the job.
-    :type display_name: str
-    :param compute: Compute definition containing the compute information for the step
-    :type compute: str
-    :param experiment_name:  Name of the experiment the job will be created under,
-        if None is provided, default will be set to current directory name. Will be ignored as a pipeline step.
-    :type experiment_name: str
+    :type component: Component
+    :param inputs: The inputs for the node. Defaults to None.
+    :type inputs: Optional[Dict[str, Union[
+        ~azure.ai.ml.entities._job.pipeline._io.PipelineInput,
+        ~azure.ai.ml.entities._job.pipeline._io.NodeOutput,
+        ~azure.ai.ml.entities._inputs_outputs.Input,
+        str,
+        bool,
+        int,
+        float,
+        Enum,
+        'Input']]]
+    :param outputs: Mapping of output data bindings used in the job. Defaults to None.
+    :type outputs: Optional[Dict[str, Union[str, ~azure.ai.ml.entities._inputs_outputs.Output, 'Output']]]
```

Comment:
I don't think it should be change because this path matches the `from azure.ai.ml.entities._inputs_outputs import Output` at the top of this file, 
The consistency is not easy to confuse subsequent research and development.